nx/test/nx/defn/tree_test.exs

```diff
@@ -6,7 +6,11 @@ defmodule Nx.Defn.TreeTest do
   doctest Nx.Defn.Tree
 
   import Nx.Defn
-  @default_defn_compiler Nx.Defn.Identity
+
+  setup do
+    Nx.Defn.default_options(compiler: Nx.Defn.Identity)
```

Comment:
In the same vein, instead of default_options, global_options or just options could work here

Reply:
The reason I went with `default` is because they can still be overridden on `Nx.Defn.jit/3` and friends.